packages/kit/src/runtime/server/page/render.js

```diff
@@ -148,8 +148,8 @@ export async function render_response({
 		}
 		// prettier-ignore
 		head += Array.from(css)
-				.map((dep) => `\n\t<link${styles.has(dep) ? ' disabled' : ''} rel="stylesheet" href="${options.prefix + dep}">`)
-				.join('');
+            .map((dep) => `\n\t<link${styles.has(dep) ? ' disabled media="screen and (max-width: 1px)"' : ''} rel="stylesheet" href="${options.prefix + dep}">`)
```

Comment:
I wonder if we can make this even simpler — any reason we can't change `media` to `none`? That way we can get rid of the `disabled` attribute too. This works for me locally, what do you reckon?

```suggestion
            .map((dep) => `\n\t<link${styles.has(dep) ? ' media="none"' : ''} rel="stylesheet" href="${options.prefix + dep}">`)
```

Reply:
Will add the changeset and tests later, sorry!

I mistakenly believed that disabled needed to be in there for Vite, don't know where I got that from!

The `media="none"` thing is a pedantic point, strictly speaking [there is no `none` value](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link#attr-media) (only `all`, `screen` and `print`), so it's relying on browsers to interpret a value they don't understand as 'don't download this'. so `screen and (max-width: 1px)` is actually "valid" and actively instructing according to spec.

That being said, I've not met a browser yet that wouldn't work as intended with `media="none"`, and I do think it's more intuitive and readable, so perhaps we should just use that?

An alternative would be `media="print"` which would prevent the browser from downloading (unless the user printed the page!), but still be spec compliant?

Reply:
Fair point — I'm not much of a purist but I did wonder about that, and it would be nice to generate HTML that validates.

`print` seems good — no real harm if the user _does_ print and download the styles. `media="(max-width: 0)"` would also work I think, but `print` is shortest.

Reply:
OK made the changes to `print`, and amended the test, however, it seems like `print` (and any non-matching media value) do still get downloaded. Just with a lowest network priority.

That's still an improvement in my book, but I retitled this PR to more accurately reflect that.